### PR TITLE
Image Signature verification istructions

### PR DESCRIPTION
**Pull request by fmount**
This patch adds the image signature verification istructions based on what is provided in the upstream documentation [1]. 
To make the directory tree consistent, the image-cache folder has been renamed.

[1] https://docs.openstack.org/glance/latest/user/signature.html

Fixes: [OSPRH-3420](https://issues.redhat.com//browse/OSPRH-3420)

### CODE REVIEW

File: config/samples/image_signature/README.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+```bash
+oc get secrets osp-secret -o jsonpath="{ .data.AdminPassword }" | base64 -d
+```
+
+- osp-secret: the secret used to store the passwords associated to the deployed
+  ctlplane
+
+
+## Known Limitations
+
```
Comment (konan-abhi) [inline config/samples/image_signature/README.md:139]: I think in known issues we should also mention failure if provide sha256 at line 95 while creating digest.

Comment (fmount) [reply config/samples/image_signature/README.md:139]: Done

**Review by konan-abhi — APPROVED** [review]
Looks good, thank you for the sample script!!

**Pull request merged**